qanary_commons/src/main/resources/queries/select_all_AnnotationOfRelation.rq

```diff
@@ -0,0 +1,16 @@
+PREFIX qa: <http://www.wdaqua.eu/qa#>
+PREFIX oa: <http://www.w3.org/ns/openannotation/core/>
+
+<urn:qanary:output> a qa:AnnotationOfRelation .
```

Comment:
@dschiese qanary_commons/src/main/resources/queries/select_all_AnnotationOfRelation.rq is no SELECT query. Wrong filename or wrong query?